src/entrypoints/recorder.ts

```diff
@@ -326,34 +342,50 @@ function initXhrObserver(cb: networkCallback, win: IWindow, options: Required<Ne
  *  NB PerformanceNavigationTiming extends PerformanceResourceTiming
  *  Here we don't care which interface it implements as both expose `serverTimings`
  */
-const exposesServerTiming = (event: PerformanceEntry): event is PerformanceResourceTiming =>
-    event.entryType === 'navigation' || event.entryType === 'resource'
-
-function prepareRequest(
-    entry: PerformanceResourceTiming,
-    method: string | undefined,
-    status: number | undefined,
-    networkRequest: Partial<CapturedNetworkRequest>,
+const exposesServerTiming = (event: PerformanceEntry | null): event is PerformanceResourceTiming =>
+    !!event && (event.entryType === 'navigation' || event.entryType === 'resource')
```

Comment:
Don't know exactly when it is better to use the helpers but maybe this is one of those times
```suggestion
    !isNull(event) && (event.entryType === 'navigation' || event.entryType === 'resource')
```

Reply:
yep... only one or two characters difference when bundled (e.g. you'd have maybe`!sQ(r)` vs `!!r`) but more intention revealing